.github/workflows/ci.yml

```diff
@@ -0,0 +1,352 @@
+name: CI 
+
+on:
+  push:
+    paths:
+      - '.github/workflows/dart_ci.yml'
+      - 'discovery_engine/**'
+      - 'discovery_engine_core/**'
+      - 'job_runner.sh'
+      - '.github/workflows/rust_ci.yml'
+      - 'discovery_engine_core/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+env:
+  DART_VERSION: '2.14.4'
+  DART_WORKSPACE: ${{ github.workspace }}/discovery_engine
+  RUST_WORKSPACE: ${{ github.workspace }}/discovery_engine_core
+  RUST_NIGHTLY: nightly-2021-09-09
+  RUSTFLAGS: "-D warnings"
+  RUSTDOCFLAGS: "-D warnings"
+
+
+jobs:
+  selection:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      dart: ${{ steps.filter.outputs.dart }}
+      rust: ${{ steps.filter.outputs.rust }}
+    steps:
+      # For pull requests it's not necessary to checkout the code
+      - uses: dorny/paths-filter@v2.10.2
+        id: filter
+        with:
+          filters: |
+            dart:
+              - '.github/workflows/dart_ci.yml'
+              - 'discovery_engine/**'
+              - 'discovery_engine_core/**'
+              - 'job_runner.sh'
+            rust:
+              - '.github/workflows/rust_ci.yml'
```

Comment:
As I understand this replaces `dart_ci.yml` and `rust_ci.yml` so they can be removed from the files which trigger job runs (and deleted, I guess).

Reply:
Yes, good point I forgot to delete them from the repository and from this path.